src/components/MaterialUI/DataDisplay/tableHelpers.test.js

```diff
@@ -601,4 +601,42 @@ describe("table helpers buildHeaderAndRowFromConfig", () => {
 		expect(rows[1].columns[1].className, "to be undefined");
 		expect(rows[1].columns[1].title, "to equal", "another 2");
 	});
+
+	it("build table rows as expected with radio", () => {
+		const changeEvent = () => console.log("just an event handler");
```

Comment:
use jest.fn() if you just need some function and don't care about implementation.